Updated Error Codes and Certification Error Tests (#182)
MewsSystems/gitbook-channel-manager-api: guidelines/responses.md

| `code` | `int` | required | Determines the type of error \(see [Error codes](#error-codes)\). |
| `message` | `string` | required | Error message with more details about the error. |
| `categoryCode` | `string` | optional\* | Category code which caused the error. This is _required_ for category errors and rate category errors. The category will be automatically unsynchronized in Mews. |
| `rateCode` | `string`  | optional\* | Rate code which caused the error. This is _required_ for rate errors and rate category errors. The rate will be automatically unsynchronized in Mews. |

> **\*Important**: If the error is a category error (code 10) or rate category error (code 11), then `categoryCode` must be specified.
> If the error is a rate error (code 9) or rate category error (code 11), then `rateCode` must be specified.

### Error codes

| Code | Description |
| :-- | :-- |
| `1` | **System error**<br>Unspecified system error. Message may be re-sent after an interval of time. |
| `2` | **Reservation not found**<br>The specified reservation could not be found in the system. When this error code is returned, it should be acknowledged and the reservation should _not_ be re-tried. |

> | `3` | **Property not found**<br>The specified property, hotel or other enterprise could not be found in the system. |

Reviewer: this is technically connection (i.e. integration), not property (hotel, enterprise).

so maybe
```suggestion
| `3` | **Connection not found**<br>The specified connection (integration) could not be found in the system. |
```
